scripts/azure/batch_node_runner_submit.sh

```
@@ -0,0 +1,40 @@
+# Batch Node Runner manual submission requires some credentials for open catalog.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)